bazel/BUILD

```diff
@@ -1,4 +1,4 @@
 exports_files([
     "pytest_wrapper.py",
-    "conftest.py"
+    "conftest.py",
```

Comment:
This wasn't caught because `format.sh` runs lints serially. An earlier lint was failing for unrelated reasons, we marked the tests as "tests-ok" and never saw this issue.